week-2-practise/level-2/2-validation.js

```diff
@@ -4,7 +4,12 @@
 // - is less than or equal to 100
 // Tip: write other small functions for each requirement
 
-function validate(num) {}
+function validate(num) {
+  if (typeof num == "number" && num > 0 && num <= 100) {
+    return "True";
+  }
+  return "False";
+}
```

Comment:
When I run this exercise, I get this result:

```
$ node week-2-practise/level-2/2-validation.js
True
True
False
False
False
```

Can you see a difference between this and the Expected Result?

```
true
true
false
false
false
```

When you see `true` and `false` in the Expected Result, it means that we want a `boolean` rather than a string. Do you know how you'd change your function above to return a `boolean` instead of a `string`?